torch/csrc/utils/python_arg_parser.cpp

```diff
@@ -413,8 +413,11 @@ bool FunctionSignature::parse(PyObject* args, PyObject* kwargs, PyObject* dst[],
       return false;
     } else if (param.check(obj)) {
       dst[i++] = obj;
+    // XXX: the Variable check is necessary because sizes become tensors when
+    // tracer is enabled. This behavior easily leads to ambiguities, and we
+    // should avoid having complex signatures that make use of it...
     } else if (allow_varargs_intlist && arg_pos == 0 && !is_kwd &&
-               THPUtils_checkIndex(obj)) {
+               (THPUtils_checkIndex(obj) || THPVariable_Check(obj))) {
```

Comment:
I'm a bit confused, what exactly are the individual elements when the tracer is enabled?  I'd expect them to be 0-dimensional long variables, which should work with THPUtils_checkIndex?

Reply:
Oh, I didn't know that! I think it used to be `checkLong` before the rebase, which is why I needed the extra check. I'll remove it in a patch.

Reply:
That sounds correct.